### PR TITLE
Add postgresql_cluster entity type

### PR DESCRIPTION
**Pull request by a1exsh**
Get the list of cluster entities first, then based on it discover the actual
database entities.
Remove guessing of service port, use the default port 5432 instead.

### CODE REVIEW

**Comment by Jan-M** [comment]
:+1:

**Comment by avaczi** [comment]
:+1:

File: zmon_agent/discovery/kubernetes/cluster.py [hunk 1]
```diff
@@ -453,28 +455,55 @@ def get_cluster_postgresdbs(kube_client, cluster_id, alias, region, infrastructu
...
+                                          port=POSTGRESQL_DEFAULT_PORT,
                                           user=postgres_user,
                                           password=postgres_pass,
                                           dbname='postgres',
                                           sslmode='require')
         for db in dbnames:
             entity = {
-                'id': '{}-{}-{}[{}]'.format(db, service.name, service.namespace, cluster_id),
-                'type': POSTGRESDB_TYPE,
+                'id': '{}-{}'.format(db, pgcluster['id']),
```
Comment (mohabusama) [inline zmon_agent/discovery/kubernetes/cluster.py:495]: I assume the cluster id includes the ``[<cluster id>]`` magic?

Comment (a1exsh) [reply zmon_agent/discovery/kubernetes/cluster.py:495]: Correct.  We generate the postgresql cluster ids with proper kube cluster id suffix.  Pleas see in `get_postgresql_clusters()` above.

**Pull request merged**